crates/holochain/src/sweettest/sweet_conductor_config.rs

```diff
@@ -72,15 +72,12 @@ impl SweetConductorConfig {
 
     /// Standard config for SweetConductors
     pub fn standard() -> Self {
-        /* TODO plumb these into kitsune2 somehow
-            .tune(|tune| {
-                tune.gossip_loop_iteration_delay_ms = 500;
-                tune.gossip_peer_on_success_next_gossip_delay_ms = 1000;
-                tune.gossip_peer_on_error_next_gossip_delay_ms = 1000;
-                tune.gossip_round_timeout_ms = 10_000;
-            })
-        */
-        let mut c = SweetConductorConfig::from(NetworkConfig::default()).tune_conductor(|tune| {
+        let network_config = NetworkConfig::default()
+            .with_gossip_initiate_interval_ms(500)
+            .with_gossip_min_initiate_interval_ms(1000)
```

Comment:
If this is set to 1 s and the initiation runs every 0.5 s, then half of the attempts will be in vain, no?

Reply:
With 2 peers yes, with 3+ peers it's fine. Prefer setting these to the same value?

Reply:
There's no harm doing that for tests, so yes. We have a fair amount of 2 peer tests.

Reply:
Makes sense, done! https://github.com/holochain/holochain/pull/4787/commits/77fe235353af6227e0e2938e1aff297d86d4ca76